buildspec/release/10changeversion.yml

```diff
@@ -26,9 +26,8 @@ phases:
                 npm version --no-git-tag-version "$VERSION" -w packages/${TARGET_EXTENSION}
                 # Call npm ci because 'createRelease' uses ts-node
                 npm ci
-            # TODO: fix createRelease (generates CHANGELOG) for "amazonq".
             - |
-                echo "$CODEBUILD_WEBHOOK_HEAD_REF" | >/dev/null grep master && npm run createRelease || true
+                npm run createRelease -w packages/${TARGET_EXTENSION}
```

Comment:
if we are doing this on feature/standalone, won't we run into https://github.com/aws/aws-toolkit-vscode/pull/4700 again?

Reply:
No unless there is some edge case I can't think of. feature/standalone only releases amazonq, so only changelogs for amazonq can be updated and committed.

I also think we should cherry-pick this (and other buildspec changes) into mainline to reduce merge-conflicts.